_config.yml

```diff
@@ -245,6 +245,10 @@ whitelist:
   - jekyll-redirect-from
 
 collections:
+  packaging:
```

Comment:
i added some infrastructure that will allow us to create content stubs for packaging content.  in the future when i figure out how to generate a rss feed of these from our website (i suppose i could use the xml but rss would be nicer) we can autopopulate this with a CI action.  For now it will be manual which is why there are a bunch of new stub pages added as jekyll collections to this pr.